### PR TITLE
Runnable version

### PR DESCRIPTION
**Pull request by oliverripps**

### CODE REVIEW

File: run.sh [hunk 1]
```diff
@@ -1,3 +1,3 @@
 #!/bin/bash
 
-docker run -dit -p 3000:3000 -e HTTPS=0 --restart always orb-server
+docker run -dit -p 3000:3000 -e HTTPS=0 --restart always --env "DB_HOST" --env "DB_PORT" --env "DB_USER" --env "DB_PWD" --env "DB_NAME" --env-file src/app/config/orb-server.env -v $(pwd)/src:/orb-server/src orb-server
```
Comment (Sammidysam) [inline run.sh:3]: No need to add `--env` if we are providing an `env-file`, no?

File: db.php [hunk 2]
```diff
@@ -0,0 +1,38 @@
+<?php
```
Comment (Sammidysam) [inline db.php:1]: Why did we include this file twice?

**Review by Sammidysam — COMMENTED** [review]
This is good enough, but a couple things are weird.

**Comment by Sammidysam** [comment]
Actually, does this work for you?  I have a hard time believing that the `db.php` file will truly work, as you did not change the logic to only require `remote.php`.  Instead, it will try to access a local database, which does not make sense since it will not exist.  I would not be surprised if this is part of why the application did not really work for you.  I would recommend adjusting the `db.php` file so that it only loads `remote.php`.  We can undo this later if actual development happens on this application.

I will test this myself after this change is made.

File: db.php (hunk 2 above)
Comment (oliverripps) [reply db.php:1]: One was empty. I deleted it.

File: run.sh (hunk 1 above)
Comment (oliverripps) [reply run.sh:3]: I took out the --env before each variable and there is a docker error.

**Comment by oliverripps** [comment]
@Sammidysam I changed db.php to only and always require remote.php. Sorry for the delay on this.

**Comment by Sammidysam** [comment]
I tested it out and almost good.  When I ran it, it did not seem to have the necessary stylesheet - all styles were messed up.  I figure there is another file that needs to be copied into the repository.

**Comment by Sammidysam** [comment]
Still very nice to have it at least running!

**Comment by Sammidysam** [comment]
Hold up - might have run the wrong branch, wow.

File: run.sh (hunk 1 above)
Comment (Sammidysam) [reply run.sh:3]: Only the `--env` preceding each variable?  There will certainly be an error if you give all of these variable names without any indication of what they mean (do you understand what each word in the command means?  Worth going over just to make sure you understand what you are writing).  I am arguing you can take out all of the following text:
`--env "DB_HOST" --env "DB_PORT" --env "DB_USER" --env "DB_PWD" --env "DB_NAME"`

**Comment by Sammidysam** [comment]
Okay this is very nice!  So sorry about running the wrong branch.  Last question before final sign-off: why do we need to have `class.Meter.php` in the root directory as well as its inner directory?  I think I wondered this both with `db.php` and this file, but it seems only `db.php` got corrected.

File: run.sh (hunk 1 above)
Comment (oliverripps) [reply run.sh:3]: got it.

**Comment by oliverripps** [comment]
@Sammidysam Should be ready to be merged now

**Pull request merged**